postfix/check.py

```diff
@@ -24,15 +24,24 @@ class PostfixCheck(AgentCheck):
     YAML config options:
         "directory" - the value of 'postconf -h queue_directory'
         "queues" - the postfix mail queues you would like to get message count totals for
+
+    Optionally we can run the check to use `postqueue -p` which is ran with set-group ID privileges,
```

Comment:
Can we grab all metrics doing this? Do you have a link to the doc detailing this behavior? It would be nice to have here + understand what our continued dependency on `sudo` may be. 

Also, please note that on a production server, it's unlikely that `postqueue` would have access to all queues by all users. Configuration would typically be set in the configuration files:

```
Postfix doesn't use unix permission feature to limit access to postqueue. 
Instead it uses parameters like ' authorized_mailq_users' in its configuration files.
```

We'd have to explain how to do this in our docs/config.